src/scancode/cli.py

```diff
@@ -382,6 +382,9 @@ def scan_event(item):
                     return ''
                 if item:
                     _scan_success, _scanned_path = item
+                    if len(_scanned_path)>20:
```

Comment:
Your spacing is not right. We use PEP8 for code standard. eg. `len(scanned_path) > 20`